### DISCUSSION TITLE
SO stop showing events

### DISCUSSION THREAD

**Discussion by sailormoonnnnn** [Security-Onion-Solutions/securityonion#12165] [2.4]
### Version

2.4.30

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Distributed

### Location

on-prem with Internet access

### Detail

the SO just stop showing logs after a few hours with the alert on influxdb "high redis memory usage" , im using SO 2.4.30

error=>{"type"=>"illegal_state_exception", "reason"=>"Pipeline processor configured for non-existent pipeline [suricata.ike]"}}

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by sailormoonnnnn** [comment]
there is around 400k unparsed redisqueue logs

<img width="687" alt="issue" src="https://github.com/Security-Onion-Solutions/securityonion/assets/86309098/d01ffaf9-3eb2-4bed-9d66-62e19b58ef0c">

**Reply by weslambert** [reply]
Could you share a few sample logs from /nsm/suricata/eve*.json, sanitizing them as necessary?

You can run a command similar to the following.

`cat eve-2024-01-12-20:01.json | jq '. | select(.event_type == "ike")'`
